doc-content/enterprise-only/admin-and-config/configuring-environment-mode-proc.adoc

```diff
@@ -0,0 +1,18 @@
+[id='configuring-environment-mode-proc']
+= Configuring the environment mode in {KIE_SERVER} and {CENTRAL}
+
+You can now set {KIE_SERVER} to run in `production` mode or in `development` mode. Development mode provides a flexible deployment policy that enables you to update existing deployment units (KIE containers) while maintaining active process instances for small changes, or to reset the deployment unit state before updating active process instances for bigger changes. Production mode is optimal for production environments, where each deployment creates a new deployment unit, as in previous releases.
+
+In a development environment, you can click *Deploy* to deploy the built KJAR file to a {KIE_SERVER} without stopping any running instances (if applicable), or click *Redeploy* to deploy the built KJAR file and stop any running instances. The next time you deploy or redeploy the built KJAR, the previous deployment unit (KIE container) is automatically updated in the same target {KIE_SERVER}.
+
+In a production environment, the *Redeploy* option is disabled and you can click *Deploy* only to deploy the built KJAR file to a new deployment unit (KIE container) on a {KIE_SERVER}.
```

Comment:
"... and clicking *Deploy*deploys only the KJAR file to a ...."?

Reply:
@pnagesha, @csherrar, I'd keep it as is, imo. What you suggest says something slightly different. We're emphasizing the the user has only the **Deploy** option in prod mode, not that that option deploys only the KJAR as in your rewrite.  We also typically want to keep the user-centric tone rather than the abstract "clicking".

Reply:
Keeping it as-is

Reply:
@csherrar, I see why @pnagesha was tripped up. Maybe "click only **Deploy** to" would clarify the ambiguity? As it stands is technically the "correct" way for using "only", but I can see why someone could get tripped up. Your call.

Reply:
I understand, how about changing "you can click *Deploy* only* to "you can only click *Deploy* to deploy..."?

Reply:
@csherrar That's what is typically not recommended, because then "only" modifies "click", as in clicking it is the only thing you can do, versus Deploy is the only thing you can click. So imo, "only" needs to immediately precede or immediately follow Deploy to be "correct". Whatever you feel is clearest though. It's your call.

Reply:
ah got it, will do that.